src/components/Contact/ContactForm.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from "react";
+import styles from "./ContactForm.module.css";
+import FormElement from "../utility/FormElement/FormElement";
+import Button from "../utility/Button/Button";
+
+const ContactForm = () => {
+  const [formData, setFormData] = useState({
+    name: { value: "", error: "", touched: false },
+    email: { value: "", error: "", touched: false },
+    companyName: { value: "", error: "", touched: false },
+    title: { value: "", error: "", touched: false },
+    message: { value: "", error: "", touched: false },
+  });
+
+  const handleChange = (inputField) => (e) => {
+    const { value } = e.target;
+    setFormData((prevState) => ({
+      ...prevState,
+      [inputField]: {
+        ...prevState[inputField],
```

Comment:
You wont need this line, as you are replacing all the 3 properties in the upcoming 3 lines

Reply:
not sure which line or if you mean all. I used the first  spread operator to maintain the previous state and the second to spread the specific input field . though the coming lines update the fields it still maintains the states for the remaining